src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -1,12 +1,24 @@
 package core.basesyntax;
 
+import java.time.DateTimeException;
 import java.time.LocalDate;
 import java.time.LocalDateTime;
 import java.time.LocalTime;
 import java.time.OffsetDateTime;
+import java.time.ZoneOffset;
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.util.Locale;
 import java.util.Optional;
 
 public class JavaDateTimeApi {
+    final DateTimeFormatter formatterParseDate = DateTimeFormatter
+            .ofPattern("yMMdd", Locale.US);
+    final DateTimeFormatter formatterForCustomDate = DateTimeFormatter
+            .ofPattern("dd MMM yyyy", Locale.US);
+    final DateTimeFormatter formatterForDate = DateTimeFormatter
+            .ofPattern("dd MMMM yyyy HH:mm", Locale.US);
```

Comment:
what are these variables? they are not constant fields at the moment

Reply:
done